packages/axios-extension/src/auth/index.ts

```diff
@@ -47,10 +48,10 @@ export function attachUaaAuthInterceptor(
         return uaa.getUserInfo(token);
     };
 
-    const oneTimeInterceptorId = provider.interceptors.request.use(async (request: AxiosRequestConfig) => {
-        token = token ?? (await getToken());
+    const oneTimeInterceptorId = provider.interceptors.request.use(async (request: InternalAxiosRequestConfig) => {
+        token = token ?? (await uaa.getAccessToken(refreshToken, refreshTokenUpdateCb));
```

Comment:
This code is causing an issue with the xsuaa flow since the credentials passed in will never be used.